app/views/internal/configs/show.html.erb

```diff
@@ -88,14 +88,19 @@
                                 placeholder: "email@staff.com" %>
               <div class="alert alert-info">Email of the staff account</div>
             </div>
-
             <div class="form-group">
-              <%= f.label :social_networks_handle %>
-              <%= f.text_field :social_networks_handle,
-                               class: "form-control",
-                               value: SiteConfig.social_networks_handle,
-                               placeholder: "thepracticaldev" %>
-              <div class="alert alert-info">Social networks handle (eg. Twitter, GitHub)</div>
+              <%= f.fields_for :social_media_handles do |ff| %>
+                <% SiteConfig.social_media_handles.keys.each do |social_media_platform| %>
+                  <div>
+                    <%= ff.label social_media_platform %>
+                    <%= ff.text_field social_media_platform,
+                                     class: "form-control",
+                                     value: SiteConfig.try(:social_media_handles)[social_media_platform],
```

Comment:
Why `try` if we already called `SiteConfig.social_media_handles` before? We can be pretty confident that `SiteConfig.social_media_handles` exist :)

Reply:
hmm i forgot to remove this code when i moved moved to using :hash which provides the defaults. Thanks for catching this!